config/ecs/ecs-default-config.yaml

```diff
@@ -22,8 +25,9 @@ processors:
 exporters:
   awsxray:
   awsemf:
-    log_group_name: 'aws/ecs/containerinsights/{ClusterName}/application/metrics'
-    log_stream_name: '{TaskId}'
+    namespace: ECS/Default
+    log_group_name: '/aws/ecs/containerinsights/application/metrics'
+    log_stream_name: 'metrics'
 
```

Comment:
Same situation here. If the value show in your log group is `undefined`, I think that is because the key is not found in the resource attribute map which statsd receiver send.  
Please check if the statsd receiver should send the `TaskId` and `ClusterName`

Reply:
Thanks, the thing is that statsD receiver does not send metric with name `TaskId` and `ClusterName` since it is does not need to be used in ECS only.